src/test/ui/oom_unwind.rs

```diff
@@ -0,0 +1,19 @@
+// compile-flags: -Z oom=panic
+// only-linux
```

Comment:
On Windows (and any OS without overcommit) this will reserve huge amounts of memory which will be unavailable for other processes to use. Linux has overcommit by default which means that the memory isn't actually reserved.

Reply:
Right, it isn't clear to me why this test wouldn't work on Windows. It seems very unlikely that the reservation would present a genuine problem given the size of chunks at which the test attempts to allocate memory.

Reply:
In the end I switched this test back to only running on Linux. It's a pain to deal with all the OS differences here.